tests/console/firewalld.pm

```diff
@@ -14,11 +14,12 @@ use strict;
 use warnings;
 use base "consoletest";
 use testapi;
-use utils 'systemctl';
+use utils qw(systemctl zypper_call);
 use version_utils 'is_tumbleweed';
 
 # Check Service State, enable it if necessary, set default zone to public
 sub pre_test {
+    zypper_call('in firewalld') unless (script_run("which firewalld") == 0);
```

Comment:
It is common that packages are installed and when they are already installed then good, is with it any problem ?